packages/plugin-analytics/src/utils/analytics.ts

```diff
@@ -0,0 +1,38 @@
+interface CommonOptions {
+	action?: string
+	url?: string
+	domain?: string
+	userID?: number | string
+	actionType?: string
+	type?: string
+	name: string
```

Comment:
Is `name` truly a common option? Isn't that the same as the `page` parameter in `ViewOptions`?